src/infinite-list/index.css

```diff
@@ -7,10 +7,6 @@ gb-infinite-list.gb-ui .gb-list__wrapper--grid,
   flex-wrap: wrap;
 }
 
-.gb-list__wrapper--grid .gb-list__item.gb-ui {
```

Comment:
This was duplicated in the list component and the infinite-list component.  Let me know if that was intentional and I can put it back in :)